flink-tests/src/test/java/org/apache/flink/test/scheduling/AdaptiveSchedulerITCase.java

```diff
@@ -72,10 +94,19 @@ private static Configuration getConfiguration() {
                             .setNumberSlotsPerTaskManager(NUMBER_SLOTS_PER_TASK_MANAGER)
                             .build());
 
+    @Before
+    public void ensureDeclarativeResourceManagement() {
+        assumeTrue(ClusterOptions.isDeclarativeResourceManagementEnabled(configuration));
```

Comment:
The `configuration` we check here is different to the one used in the `MiniCluster`. There is no real point having the member `configuration` here, I guess.